gradle/publish-runnable.gradle

```diff
@@ -34,7 +34,7 @@ bintray {
     publish = true
     pkg {
         repo = 'releases'
-        name = 'groovy-language-server'
+        name = project.ext.get("applicationName")
```

Comment:
for `project.ext` I think you need to actually declare the extra property inside each build.gradle.
`project.name` should work